dot/telemetry/telemetry.go

```diff
@@ -54,13 +61,16 @@ type KeyValue struct {
 
 var (
 	once            sync.Once
-	handlerInstance *Handler
+	handlerInstance Instance
+
+	// Enabled flag to determine if telemetry is enabled
+	Enabled = true // enabled by default
```

Comment:
make `Enabled` private.  Shouldn't be able to change this from outside the package.

Reply:
I want to disable telemetry from outside the package (i.e. in node.go line 343).  I'm following suggestion by @noot above.

Reply:
yeah I think this is fine, we could to create some `Initialise` function that accepts the `enabled bool` and is also a singleton and change this package var to internal. what do you think @timwu20

Reply:
Yup that works for me.

Reply:
Sounds good, I believe I've implemented it correctly now.